lib/cancel-stuck-transactions.js

```diff
@@ -0,0 +1,66 @@
+import * as Sentry from '@sentry/node'
+import { StuckTransactionsCanceller } from 'cancel-stuck-transactions'
+import ms from 'ms'
+import timers from 'node:timers/promises'
+
+const ROUND_LENGTH_MS = ms('20 minutes')
+const CHECK_STUCK_TXS_DELAY = ms('1 minute')
+
+export const createStuckTransactionsCanceller = ({ pgClient, signer }) => {
+  return new StuckTransactionsCanceller({
+    store: {
+      async set ({ hash, timestamp, from, maxPriorityFeePerGas, nonce }) {
+        await pgClient.query(`
+          INSERT INTO transactions_pending (hash, timestamp, from_address, max_priority_fee_per_gas, nonce)
+          VALUES ($1, $2, $3, $4, $5)
+          `,
+        [hash, timestamp, from, maxPriorityFeePerGas, nonce]
+        )
+      },
+      async list () {
+        const { rows } = await pgClient.query('SELECT * FROM transactions_pending')
+        return rows.map(row => ({
+          hash: row.hash,
+          timestamp: row.timestamp,
+          from: row.from_address,
+          maxPriorityFeePerGas: row.max_priority_fee_per_gas,
+          nonce: row.nonce
+        }))
```

Comment:
Nitpick, feel free to ignore.

If the only reason for this `rows.map` function is to rename the columns, then I think the following would be slightly better, as the database will complain you reference a column that does not exist. Notice that I am enclosing the final column names in double quotes to prevent PG from converting them to lowercase (plus `from` is likely a reserved word?).

```suggestion
        const { rows } = await pgClient.query(`
          SELECT 
            hash, 
            timestamp, 
            from_address as "from",
            max_priority_fee_per_gas as "maxPriorityFeePerGas",
            nonce
          FROM transactions_pending
        `)
        return rows
```

Reply:
good idea!